### PR TITLE
Correct tiering and hoist alerts

### PR DESCRIPTION
**Pull request by matthewisabel**
## What?
This adds our alerts component to the top of the page for integrations and features and also corrects inaccurate tiering.

## Why?
Better docs, correct info

## How?
Used new component

### CODE REVIEW

File: docs/beacons.mdx [hunk 1]
```diff
@@ -3,6 +3,13 @@ sidebar_position: 5
 title: Beacons
 id: beacons
 ---
+import Alert from "../src/components/Alert";
+
+<Alert alertType="info">
+  Beacons is available to organizations on the {` `}
```
Comment (jsani-radar) [inline docs/beacons.mdx:9]: we don't really reference the concept of an "organization" externally, and this fields a little wordy - any opposition to just saying "Beacons is available on the Enterprise plan"?

File: docs/faqs.mdx [hunk 2]
```diff
@@ -107,6 +109,11 @@ If you need assistance, you can also forward requests to your customer success m
 Use the appropriate role (owner, admin, write, or read) for each co-worker's account. See [Radar security best practices](#what-are-security-best-practices-for-radar).
 
 ### How do data access controls work?
+<Alert alertType="info">
+  Data accesss controls is available to organizations on the {` `}
```
Comment (jsani-radar) [inline docs/faqs.mdx:113]: nit: should this be "are" instead of "is", as there are controls for multiple entities?

**Review by jsani-radar — APPROVED** [review]
lgtm overall - two small nits on copy

**Comment by matthewisabel** [comment]
Changed both, @jsani-radar you 👍 ?

File: docs/beacons.mdx [hunk 3]
```diff
@@ -42,7 +49,6 @@ Beacons also provides the following events:
 
 You can receive events client-side via the [SDK](/sdk) or server-side via [event integrations](/integrations), including webhooks.
 
-Beacons is available on the [Team](https://radar.com/pricing) plan and higher.
```
Comment (jsani-radar) [inline docs/beacons.mdx:52]: ah I think this was the extra `\n` I was referring to - now there's 2 before "how it works"

**Pull request merged**